setup.py

```diff
@@ -42,8 +42,8 @@
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3",
     ]
-    + [("Programming Language :: Python :: %s" % x) for x in "2.7 3.5 3.6 3.7 3.8".split()],
-    install_requires=["glob2", "Mako", "parse", "parse_type", "py", "pytest>=4.3", "six>=1.9.0"],
+    + [("Programming Language :: Python :: %s" % x) for x in "3.6 3.7 3.8 3.9".split()],
+    install_requires=["glob2", "Mako", "parse", "parse_type", "py", "pytest>=4.3"],
```

Comment:
we should also add a 
```python
python_requires=">=3",
```

Can you also add an entry in the CHANGES.rst file, under a new "Unreleased" release?
Like this:
```
Changelog
=========

Unreleased
-----------
- Drop compatibility for python 2 and officially support only python >= 3.6.

4.0.2
-----
```

Reply:
Thnx